src/storage/storage_info.cpp

```diff
@@ -2,6 +2,6 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 31;
+const uint64_t VERSION_NUMBER = 32;
```

Comment:
Is the storage version increment required?

Reply:
Only adding a new type should not break the storage version, could you try again and check what the error is (if there is still an error)?